app/models/plan.rb

```diff
@@ -9,4 +9,8 @@ class Plan < ActiveRecord::Base
   validates_presence_of :plan_code, :name, :amount, :payment_methods
 
   has_and_belongs_to_many :projects, join_table: 'projects_plans'
+
+  def formatted_amount
+    amount / 100
```

Comment:
The value is returned in cents, @IgorMarques . So, to format for reais I need to divide for 100, right?